CppCoreGuidelines.md

```diff
@@ -9567,11 +9540,27 @@ The `make_shared()` version mentions `X` only once, so it is usually shorter (as
 
 ##### Reason
 
-For convenience and consistency with `shared_ptr`.
+`make_unique` gives a more concise statement of the construction.
+It also ensures exception safety in complex expressions.
 
-##### Note
+##### Example
+
+    unique_ptr<Foo> p {new Foo{7}};    // OK: but repetitive
 
-`make_unique()` is C++14, but widely available (as well as simple to write).
+    auto q = make_unique<Foo>(7);      // Better: no repetition of Foo
+
+    // Not exception-safe: the compiler might interleave the computations of arguments as follows:
```

Comment:
I think this is not a concern since C++17 changed function args from unsequenced to indeterminately-sequenced. Should we contrive something with a built-in operator or just drop this part of the example?

Reply:
If you were writing a library you might want it to support pre-c++17 compilers at which point this would still be relevant advice. Not sure if that's a use case the core guidelines intend to cater to.